src/libsync/discovery.cpp

```diff
@@ -14,6 +14,9 @@
 
 #include "discovery.h"
 #include "common/syncjournaldb.h"
+#ifdef Q_OS_WIN
+#include "common/filesystembase.h"
+#endif
```

Comment:
I guess @er-vin would argue that it is not needed and I think too that conditional compilation should be reduced to a minimum.

Reply:
@FlexW Yes. In fact, I am not sure myself. It's just because I put a condition on a function call from this header, I thought it would look more consistent to put conditionals in both places just to emphasize that I want this behavior ONLY for Windows. But, I can remove this ifdef, not really fighting for it to be here.